reference/types/string.md

```diff
@@ -1,13 +1,25 @@
 # String
 
+## The Concept
+
 A string is a sequence of [characters][type-char] (letters, digits, punctuation, etc.).
 
-# Teaching tactics
+## What to cover
+
+Tell a student what a string is in your language, how strings can be manipulated, and ensure they understand where to look for docs on strings.
+
+- **Explain what strings are in your language?** Is it a sequence of bytes, UTF-8 characters or something else? Are strings objects or primitives?
+- **How are strings brought together?** Do you join strings, add strings, concatenate strings, interpolate strings? It's worth giving a hint as to which is normal, or letting the user know all are ok!
+- **How are strings split apart?** How do I extract a bit of a string?
+- **Where do helper methods for strings live?** If I want to trim a string, or check it's length, how do I do that in your language? Are these string methods or functions that act on strings?
+
+## Exercises
 
-- Be explicit about what a string represents. Is it a sequence of bytes, UTF-8 characters or something else?
+### Log Lines
 
-# Implementations
+This exercise covers joining, triming, case-changing, and substring extraction. It is a great exercise for covering the most basic elements of Strings, and ensuring a student finds the docs for strings.
```

Comment:
Should that that last "strings" be capitalized like the first "Strings" in this sentence?

Reply:
I would have thought the opposite; all mention of strings should use lower case "s". AFAIK, strings with upper case "S" indicates an object (please correct me if this was not the intended implication here). Not all languages that have the concept of strings, have such objects.

Reply:
That was my question... if the first is meant to be capitalized, then isn't the second?  Or are both meant to be lower-cased?

Reply:
It should probably be lower case consistently.
```suggestion
This exercise covers joining, triming, case-changing, and substring extraction. It is a great exercise for covering the most basic elements of strings, and ensuring a student finds the docs for strings.
```